os/drivers/pm/pm.c

```diff
@@ -128,6 +130,9 @@ static int pm_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 			}
 		}
 		break;
+	case PMIOC_SUSPEND_COUNT:
+		ret = pm_suspendcount((int)arg);
```

Comment:
How about using return for only error checking?

Reply:
`pm_check_domain` already setting `EINVAL` https://github.com/Samsung/TizenRT/blob/master/os/pm/pm_domain_register.c#L62 and we are returning it from `ioctl` call https://github.com/Samsung/TizenRT/blob/master/os/drivers/pm/pm.c#L147